lib/TypescriptCompiler.js

```diff
@@ -227,15 +227,6 @@ function concatExternalJS(visualPackage) {
     });
 }
 
-function createTypingsLink(visualPackage) {
```

Comment:
I haven't found any usage, so, I decided to remove it in order to keep the clean code.